R/gmailr.R

```diff
@@ -25,8 +25,12 @@ NULL
 NULL
 
 the <- new.env(parent = emptyenv())
-the$id <- "955034766742-huv7d1b1euegvk5vfmfq7v83u4rpdqb0.apps.googleusercontent.com"
-the$secret <- "rpJPeEMnDOh7qNAVjUh_aKlO"
+
+.onLoad <- function(libname, pkgname) {
```

Comment:
I think we should move this to zzz.R.

Reply:
OK, sure. For the first pass I was trying to change as little as possible to make the changes easier to see. Locally, I have already moved all the auth into its own file in subsequent commits. I can also move the `.onLoad()` to `zzz.R`.

Reply:
done